### PR TITLE
Entities Created

### PR DESCRIPTION
**Pull request by coeus-ashfaq**
All necessary entities are created.

### CODE REVIEW

**Comment by coeus-ashfaq** [comment]
All necessary entities are created.

File: src/AppBundle/Entity/Department.php [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * @var string
+     * @ORM\Column(name="name", type="string", length=255)
+     */
+
```
Comment (mrehanali) [inline src/AppBundle/Entity/Department.php:28]: Remove extra spaces here.

File: src/AppBundle/Entity/Department.php [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    private $name;
+
+    /**
+     * @var string
+     * @ORM\Column(name="location", type="string", length=255)
+     *
+     */
+
```
Comment (mrehanali) [inline src/AppBundle/Entity/Department.php:36]: extra spaces.

File: src/AppBundle/Entity/Department.php [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    private $location;
+
+
+    /**
+     * @return int
+     */
+
```
Comment (mrehanali) [inline src/AppBundle/Entity/Department.php:43]: extra spaces.

Comment (mrehanali) [reply src/AppBundle/Entity/Department.php:43]: Remove extra spaces from throughout the code.

**Pull request merged**